### PR TITLE
Add very much needed documentation for tools/run_tests/ directory

### PR DESCRIPTION
**Pull request by jtattermusch**
This is a documentation-only change.

### CODE REVIEW

File: tools/run_tests/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+######Useful options
+- `--regex` use regex to select particular scenarios to run.
+
+#Stress tests (run_stress_tests.py)
+
+Runs modified interop tests clients and servers under heavy load for an extended period of time to discover potential stability issues.
+The tests are internally using Kubernetes to run the client and server on GKE and upload statistics to BigQuery.
+
+Run `tools/run_tests/run_stress_tests.py --help` for more details.
```
Comment (sreecha) [inline tools/run_tests/README.md]: The tool's name is `run_on_gke.py and it is under `tools/run_tests/stress_test` directlry. Can you use the following text instead?  

"
`tools/run_tests/stress_test/run_on_gke.py --gcp_project_id=<google-cloud-platform-project-id> --config_file=<path-to-config-file>` 

The directory `tools/run_tests/stress_test/configs/` contains the config files for several scenarios
"

**Comment by jtattermusch** [comment]
Addressed the comment and used the text you provided.

**Comment by sreecha** [comment]
LGTM. Doc bug can be merged - once you take care of that CLA stuff.

**Comment by jtattermusch** [comment]
CLA is fine, I'm a googler. Merging.

**Pull request merged**